crates/core/component/auction/src/state_key.rs

```diff
@@ -8,11 +8,11 @@ pub mod parameters {
     }
 }
 
-pub mod store {
+pub mod auction_store {
     use crate::auction::id::AuctionId;
 
     pub fn prefix() -> &'static str {
-        "auction/store/"
+        "auction/auction_store/"
```

Comment:
i might mark this as `consensus-breaking` to be formal, even if the auction component isn't technically live yet

Reply:
i hear you, but i think the tag is more useful if it lets us get a filtered view of what is actually consensus compatible vs what isn't